City-of-Helsinki/hitas
# Frontend: Search, modify & add property managers

# Hitas Pull Request

# Description

New features:
- Search, modify and add property managers in the codes view
- Modify property manager in the apartment details view and in the housing company view

Modification:
- Save button works as back button when the data is unmodified in owner and manager modification modals

## Pull request checklist

Check the boxes for each DoD item that has been completed:

- **Testing**
- **Database**
- **Documentation**

## Test plan

- Modify the property manager info in the apartment details and housing company views, and check that the validations work as expected and the data updates after modification.
In the property managers tab in codes view:
- Search property managers by name and/or email and check you can modify the selected property manager and the data updates after modification.
- Add new property manager and check it is added in the property managers list

## Tickets

This pull request resolves all or part of the following ticket(s): HT-640, HT-644
